src/utils/query.test.ts

```diff
@@ -39,76 +39,11 @@ describe('API utils', () => {
                 });
             });
             describe('should correctly parse data', () => {
-                it('should parse modern schema result to KeyValueRow', () => {
```

Comment:
It would be good, if you add some tests for multi schema.

Actually `parse modern schema` tests could be used with little changes to test `parseResult` function

Reply:
added tests